tests/covalent_dispatcher_tests/_core/execution_test.py

```diff
@@ -294,17 +318,15 @@ def workflow(x):
     assert result_object._error == "The following tasks failed:\n0: failing_task"
 
 
-async def test_run_workflow_does_not_deserialize(mocker):
+@pytest.mark.asyncio
+async def test_run_workflow_does_not_deserialize(mocker, test_cluster, event_loop):
     """Check that dispatcher does not deserialize user data when using
     out-of-process `workflow_executor`"""
 
-    from dask.distributed import LocalCluster
-
     from covalent._workflow.lattice import Lattice
     from covalent.executor import DaskExecutor
 
-    lc = LocalCluster()
-    dask_exec = DaskExecutor(lc.scheduler_address)
+    dask_exec = DaskExecutor(test_cluster.scheduler_address)
```

Comment:
Let's switch to a local executor, less stuff to handle.

Reply:
Agreed. The original use for Dask was to run the tasks out-of-process, but LocalExecutor now does  that too.

Reply:
Thanks, I will switch to using a local executor...